Xero.NetStandard.OAuth2/Client/ApiException.cs

```diff
@@ -69,14 +69,15 @@ public ApiException(int errorCode, string message, dynamic errorContent = null)
         }
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="ApiException"/> class.
+        /// Initializes a new instance of the <see cref="ApiException"/> class
+        /// for the case when a rate limit is hit
         /// </summary>
         /// <param name="errorCode">HTTP status code.</param>
         /// <param name="message">Error message.</param>
         /// <param name="errorContent">Error content.</param>
-        /// <param name="limitType">Error content.</param>
-        /// <param name="retryAfter">Error content.</param>
-        public ApiException(int errorCode, string message, int retryAfter, dynamic errorContent = null, string limitType = null) : base(message)
```

Comment:
As a stylistic choice, I would argue against sandwiching the `errorContent` argument between two new ones. Keep it in the same absolute position as the other ctor (ie after `message`), or the same relative position (ie last)